Vagrantfile.centos7

```diff
@@ -12,7 +12,7 @@ Vagrant.configure("2") do |config|
     v.cpus = 2
   end
   config.vm.provision "shell", inline: <<-SHELL
-    set -e -u -o pipefail
+    set -Eeuxo pipefail
```

Comment:
why do you need `-E`?

Reply:
or `-x`? This script seems trivial, I don't see a need to have a trace

Reply:
All of the `yum` commands are done with `-q` so while testing I wasn't sure if the package I requested was actually being installed. I can drop it, but the provisioning commands being really quiet for `Vagrantfile.centos7` confused me when testing.